daml-lf/scenario-interpreter/BUILD.bazel

```diff
@@ -35,3 +40,30 @@ da_scala_test_suite(
         "//daml-lf/language",
     ],
 )
+
+daml_compile(
+    name = "CollectAuthority",
+    srcs = ["src/perf/resources/damls/CollectAuthority.daml"],
+    visibility = ["//visibility:public"],
+)
+
+da_scala_benchmark_jmh(
+    name = "scenario-perf",
+    srcs = glob(["src/perf/benches/**/*.scala"]),
+    data = [
+        ":CollectAuthority.dar",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//bazel_tools/runfiles:scala_runfiles",
+        "//daml-lf/archive:daml_lf_archive_reader",
+        "//daml-lf/archive:daml_lf_dev_archive_java_proto",
+        "//daml-lf/data",
+        "//daml-lf/interpreter",
+        "//daml-lf/language",
+        "//daml-lf/scenario-interpreter",
+        "//daml-lf/transaction",
+        "@maven//:com_google_protobuf_protobuf_java",
+        "@maven//:org_typelevel_paiges_core_2_12",
```

Comment:
```suggestion
        "@maven//:ch_qos_logback_logback_classic",
        "@maven//:org_typelevel_paiges_core_2_12",
```

Reply:
I assume this pulls in some logger and if we don't do this, logging is a no-op. Isn't the latter what we want to avoid noise?

Reply:
This is to avoid noise in the output of the test 
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

The engine does not log anything, there should not have any impact.